omero/developers/Clients/ImportLibrary.txt

```diff
@@ -55,7 +55,7 @@ data files are no longer duplicated and any nested directory structure is
 preserved. It also allows OMERO to take advantage of pre-generated pyramids
 available in some formats e.g. SVS.
 
-For full details of the import workflow see :doc:`/developers/ManagedRepository/ManagedRepository`.
+For full details of the import workflow see :doc:`/developers/ImportFS`.
```

Comment:
the other issue is we now have links labelled 'Managed Repository' which go to a file titled 'Import under OMEROfs'. Simplifying the page url/location/title is good but we probably want to ensure we still have managed repo in the title or first line of the page.